.travis.yml

```diff
@@ -19,6 +19,9 @@ script:
 - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
     cargo bench --verbose --no-run;
   fi
+- if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
```

Comment:
After adevore/rudy#16 is completed, I plan on doing a 0.1 release of Rudy. My goal is to do a release this weekend.